packages/utils/src/types.ts

```diff
@@ -957,7 +957,7 @@ export interface SchemaUtilsType<
   getDefaultFormState(
     schema: S,
     formData?: T,
-    includeUndefinedValues?: boolean
+    includeUndefinedValues?: boolean | "once"
```

Comment:
Also, update the function docs to add details for the new option

```suggestion
    includeUndefinedValues?: boolean | "noRecursion"
```